models/certs.go

```diff
@@ -32,3 +34,51 @@ type CertInfo struct {
 	ValidTo   string    `json:"ValidTo"`
```

Comment:
Your final answer must be the great and the most complete as possible, it must be outcome described.